apps/cards/app.js

```diff
@@ -14,6 +14,22 @@
 Bangle.loadWidgets();
 Bangle.drawWidgets();
 
+// get brightness
+let brightness;
+
+function loadBrightness() {
+  try {
+      const d = require('Storage').readJSON("setting.json", 1) || {};
+      brightness = Object.assign({
+          'brightness': 0.1
+      }, d || {});
```

Comment:
To save on holding all of settings in memory, we could do:

```javascript
const settings = require('Storage').readJSON("setting.json", 1) || {};
brightness = settings.brightness || 0.1;
```

And use just `brightness` below

Reply:
Sure sounds good to me. Did not do much too developing until now. Tips are very welcome, thanks :) You cannot however use the variable name `settings` as it is already used for the settings of the cards app itself.

Reply:
Cool, that's fair, we could call it whatever you like - give me a shout if you have any trouble making the changes